project/Build.scala

```diff
@@ -24,7 +24,7 @@ object ApplicationBuild extends Build {
   }
 
   def isPlay24: Boolean = {
-    playMajorAndMinorVersion == "2.4"
+    playMajorAndMinorVersion == "2.5"
```

Comment:
I guess that this change will break Play 2.4 builds because now there is no way for sbt to distinguish whether the build is targeted to Play 2.4 or 2.5.
Would you please confirm that it works for both 2.4 and 2.5?